src/assets/js/main.js

```diff
@@ -1,194 +1,199 @@
+        return
+    }
+    if($('#phoneInput').val().trim().length < 10){
+        $('#phoneInput').css('border','3px solid rgba(203, 68, 75, 0.8)');
+        return
+    }
+    return true;
+}
 
-    userRowName.innerText = userName.value;
-    userRowEmail.innerText = userEmail.value;
-    userRowAddress.innerText = userAddress.value;
-    userRowPhone.innerText = userPhone.value;
-    //_________________________________________
-    userName.value = '';
-    userEmail.value = '';
-    userAddress.value = '';
-    userPhone.value = '';
-    closeForm();
+function createNewElement(){
+    let randomID = String('userID-' + Math.random() * 1000000).split('.')[0];
+    setDefaultInputBorders();
+    if(checkInputValuesRequires() == true){
+        $('<tr>',{
+            id : randomID,
+            class : 'userRow'
+        }).appendTo($('#inputDataArea'));
+        $('<td>',{
+            class : 'userCheckbox'
+        }).appendTo($('#' + randomID));
+        $('<td>',{
+            class : 'userName'
+        }).appendTo($('#' + randomID));
+        $('<td>',{
+            class : 'userEmail'
+        }).appendTo($('#' + randomID));
+        $('<td>',{
+            class : 'userAddress'
+        }).appendTo($('#' + randomID));
+        $('<td>',{
+            class : 'userPhone'
+        }).appendTo($('#' + randomID));
+        $('<td>',{
+            class : 'userButtonsArea'
+        }).appendTo($('#' + randomID));
+    
+        $('<input>', {
+            type : 'checkbox',
+            class : 'checkboxData'
+        }).appendTo($('#' + randomID + ' .userCheckbox'));
+        
+        $('<button>',{
+            class : 'editBtn',
+            'data-title' : 'Edit'
+        }).appendTo($('#' + randomID + ' .userButtonsArea'))
+        $('<button>',{
+            class : 'deleteBtn',
+            'data-title' : 'Delete'
+        }).appendTo($('#' + randomID + ' .userButtonsArea'))
+    
+        $('<i>',{
+            class : 'fas fa-pen'
+        }).appendTo($('#' + randomID + ' .editBtn'))
+        $('<i>',{
+            class : 'fas fa-trash-alt'
+        }).appendTo($('#' + randomID + ' .deleteBtn'))
+    
+        $('#' + randomID + ' .editBtn').on('click', function(){
+            editButtonClick(randomID);
+        })
+        
+        $('#' + randomID + ' .deleteBtn').on('click', function(){
+            deleteUserRow(randomID);
+        })
+    
+        $('#' + randomID + ' .userName').text($('#nameInput').val());
+        $('#' + randomID + ' .userEmail').text($('#emailInput').val());
+        $('#' + randomID + ' .userAddress').text($('#addressInput').val());
+        $('#' + randomID + ' .userPhone').text($('#phoneInput').val());
+    
+        closeForm();
+    }
 }
 
-// _________key event_________
-window.addEventListener('keyup', function(e){
-    {
-        if(e.keyCode == 27){
+// _________key events_________
+$(window).on('keyup', function(e){
```

Comment:
You binding an event for the whole window is it makes sense? 
Maybe it will be better to do for form use to submit/escape

Reply:
resolved